doc/spark-standalone.md

```diff
@@ -0,0 +1,164 @@
+
+### spark
+Provisions and manages a spark node running over the built in resource manager (Standalone Mode)
+
+#### Variables
+
+|Name                                 |Default         |Description                                                                  |
+|:------------------------------------|:--------------:|:----------------------------------------------------------------------------|
+|hadoop_profile                       |(without-hadoop)|Hadoop profile (e.g.,  hadoop2.4, hadoop2.6, etc)                            |
+|slave_group                          |(all)           |Hosts where executors will be launched                                       | 
```

Comment:
Let's stick with `spark_slave_ansible_group` for consistency with the other roles.